plugins/clamd.js

```diff
@@ -299,11 +299,11 @@ exports.hook_data_post = function (next, connection) {
             // The current host returned an unknown result.  If other hosts are available,
             // then try those before returning a DENYSOFT.
             if (hosts.length) {
-                connection.logwarn(plugin, 'unknown result: "' + result + '" from host ' + host);
+                connection.logwarn(plugin, `unknown result: "${result}" from host ' + ${host}`);
```

Comment:
You've got a stray ' in there (and below)

Reply:
Just pushed the version with the stray '